main/FatesHistoryInterfaceMod.F90

```diff
@@ -1145,10 +1160,12 @@ subroutine set_history_var(this, vname, units, long, use_default, avgflag, vtype
                                            ! explict name (for fast reference during update)
                                            ! A zero is passed back when the variable is
                                            ! not used
+    
 
     ! locals
     integer :: ub1, lb1, ub2, lb2    ! Bounds for allocating the var
     integer :: ityp
+    integer :: nonfates              ! non-optional 
```

Comment:
It looks like this doesn't currently have any specified use, correct?

Reply:
yes, I think that is an artifact and should be removed